jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -291,4 +343,28 @@ public boolean matches(String path)
                 return false;
         }
     }
+
+    public static class ServletMatchedPath implements MatchedPath
+    {
+        private final String servletName;
+        private final String pathInfo;
+
+        public ServletMatchedPath(String servletName, String pathInfo)
+        {
+            this.servletName = servletName;
```

Comment:
this isn't the servletName, it is the pathMatch!

Reply:
Done, using MatchedPath.from() instead.